docs/en/README.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+## Requirements
+
+- The minimum version of PHP required by this package is `8.0`.
+- `PDO` PHP extension.
+
+## Installation
+
+The package could be installed with composer:
+
+```
+composer require yiisoft/cache-db --prefer-dist
+```
+
+## Migration
+
+The package provides a migration that creates the cache table for default `{{%cache}}`. You can use it as follows:
```

Comment:
I think - need add raw sql files for all dbms, for direct create tables and add this to readme